var/spack/repos/builtin/packages/py-cdat-lite/package.py

```diff
@@ -40,3 +40,14 @@ class PyCdatLite(PythonPackage):
     depends_on("python@2.5:2.8", type=('build', 'run'))
     depends_on("py-numpy", type=('build', 'run'))
     depends_on('py-setuptools', type='build')
+
+    phases = ['install']
+
+    def install(self, spec, prefix):
+        """Install everything from build directory."""
+        install_args = self.install_args(spec, prefix)
+        # Combine all phases into a single setup.py command,
+        # otherwise extensions are rebuilt without rpath by install phase:
```

Comment:
Hmm, I don't know if `--skip-build` does what I think it does. According to the help message:
```
$ python setup.py install --help
...
  --skip-build                         skip rebuilding everything (for
                                       testing/debugging)
```
I figured that this meant I could run:
```
$ python setup.py build
$ python setup.py install --skip-build
```
and separate out the logic, but this doesn't work at all for `py-numpy`. The build phase doesn't actually do much of anything, and the install phase crashes when I use `--skip-build`. So I won't be adding `--skip-build`. Guess I'll have to investigate this a bit more.